### ISSUE TITLE
Not possible to set cookie encode option

### ISSUE THREAD

**Issue by thomasmarr** [expressjs/session#883]
One of the options which can be set on `cookie.serialize` is `encode` [see here](https://github.com/jshttp/cookie#encode)

In `express-session` it is not possible to set this option:
- https://github.com/expressjs/session/blob/5df613c481bc7c5979aeaeac691b64ef0a5c4948/index.js#L161
- https://github.com/expressjs/session/blob/5df613c481bc7c5979aeaeac691b64ef0a5c4948/index.js#L243
- https://github.com/expressjs/session/blob/5df613c481bc7c5979aeaeac691b64ef0a5c4948/index.js#L657
- https://github.com/expressjs/session/blob/5df613c481bc7c5979aeaeac691b64ef0a5c4948/session/cookie.js#L117

I believe express-session should expose this, as the default `encodeUriComponent` is not _always_ the desired behaviour. I would be willing to try and put a PR together for this if it was likely to be merged promptly.

**Comment by FlinthDevs** [comment]
I agree, in my context, cookies are urlEncoded once more by a mechanism I have no control on and I had to patch the module to allow encode to be modified same as other ones.
